add whitespace analyzer docs (#8531)
opensearch-project/documentation-website: _analyzers/whitespace.md

---
layout: default
title: Whitespace analyzer
nav_order: 60
---

# Whitespace analyzer

> The `whitespace` analyzer breaks text into tokens based only on whitespace characters (spaces, tabs, and others). It does not apply any transformations, such as lowercasing or removing stopwords, therefore the case of the original text is retained and punctuation is included as part of the tokens.

Reviewer: ```suggestion
The `whitespace` analyzer breaks text into tokens based only on white space characters (for example, spaces and tabs). It does not apply any transformations, such as lowercasing or removing stopwords, so the original case of the text is retained and punctuation is included as part of the tokens.
```
